app/views/pages/home.html.haml

```diff
@@ -26,25 +26,24 @@
     %section.content-section
       .grid-row
         .column-one-half
-          %h2.heading-medium.content-section__title Use and share government data
-          %p GOV.UK Registers helps you to:
-          %ul.list.list-bullet
-            %li build and maintain your service with accurate data that is updated for you
-            %li reduce the time and cost of sourcing data from across government
-            %li easily share data with other government departments
+          %h2.heading-medium.content-section__title What is a register?
+          %p A register is a list of information that is managed and approved by government.
+          %p For example, the country register lists all countries recognised by the Foreign and Commonwealth Office.
+          %p You can use registers to build your service.
+          %p The UK Government and Parliament petitions service uses the country register to help users select their country from a dropdown menu, for instance.
```

Comment:
Remove 'Government' so it reads 'The UK Parliament petitions service...'

Reply:
Done.